modules/samples/default/decision-tree/README.MD

```diff
@@ -7,7 +7,9 @@ Algorithm: Decision-Tree
 Before Running
 ==============
 
-* Start WSO2 Machine Learner product with a fresh database.
+* Start WSO2 Machine Learner product.
+
+* Make sure jq is installed.
```

Comment:
Added the link in readme files : https://github.com/wso2/product-ml/commit/81c096aa2e69d968d530df9a2cc08e8cbca72847